src/main/kotlin/com/gamenews/Application.kt

```diff
@@ -12,6 +14,8 @@ fun main() {
 }
 
 fun Application.module() {
-    configureRouting()
+    val controller: Controller = Controller(ArticlesDatabase())
```

Comment:
Let's split this into two separate lines.

Reply:
For example:
```
val articlesDatabase = ArticlesDatabase()
val controller = Controller(articlesDatabase)
```

Reply:
Done